doc/docs/modules/ROOT/pages/quickstart.adoc

```diff
@@ -8,28 +8,274 @@ Get started fast for common scenarios, using Neo4j Streams plugin or Kafka Conne
 
 == Neo4j Spark Connector Plugin
 
-=== Install the Plugin
+=== Overview
 
-* Download the latest release jar from https://github.com/neo4j-contrib/neo4j-spark-connector/releases
-* (Installation instructions here)
+Neo4j Spark Connector allows Spark to read from and write to Neo4j databases.
 
-=== Spark Settings
+=== Getting Started
 
-(Setting detail here)
+Reading all the nodes of type `Person` from your local Neo4j instance is as simple as this:
+
+```scala
+val df = spark.read.format("org.neo4j.spark.DataSource")
+        .option("url", "bolt://localhost:7687")
+        .option("labels", "Person")
+        .load()
+```
+
+=== Neo4j Options
+
+Spark method `option` is used to configure the Neo4j Connector.
+
+```scala
+val df = spark.read.format("org.neo4j.spark.DataSource")
+        .option("url", "bolt://localhost:7687")
+        .option("authentication.type", "basic")
+        .option("authentication.basic.username", "myuser")
+        .option("authentication.basic.password", "neo4jpassword")
+        .option("labels", "Person")
+        .load()
+```
 
 .Most Common Needed Configuration Settings
 |===
-|Setting Name |Description |Default Value
+|Setting Name |Description |Default Value |Required 
+
+4+|*Driver Options*
+
+|`url`
```

Comment:
I'm not sure about this one.  Normally in driver URLs you can specify the database, neo4j://localhost/foo

We may want to do it this way and entirely drop the database option, WDYT?   If we *don't* drop the database option, then we have to figure out what to do:

```
.option("url", "neo4j://localhost/foo")
.option("database", "bar")
```

Reply:
I wasn't aware of this, in fact here we used here the same approach as streams. I tried to look into the Java Driver docs but I didn't find anyting.
If you agree I would like to keep the same approach as the Kafka Connect Sink, so the behaviour in the case that you highlighted:
```
.option("url", "neo4j://localhost/foo")
.option("database", "bar")
```

Will use the bar as database `bar`. Otherwise will use `foo` so the explicit option has the "priority" respect to the URL. We can document it.
WDYT?